<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetPassWindow</class>
 <widget class="QDialog" name="ResetPassWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reset Password</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select account to reset it's password</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="MLineEdit" name="lineEdit" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
   <property name="frame" stdset="0">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="MLabelButton" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MLabelButton" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MLineEdit</class>
  </customwidget>
  <customwidget>
   <class>MLabelButton</class>
   <extends>QLabel</extends>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
